### DISCUSSION TITLE
a NUC ("mini-PC) solely for mpv video playback on Linux

### DISCUSSION THREAD

**Discussion by user-1-2-3-4-5** [mpv-player/mpv#14985] [Q&A]
Can you estimate what type of a NUC computer is fully capable for mpv playback on Linux (Fedora)? Is a 12th-generation Intel processor with 16 GB of DDR4 RAM enough?

I play up to 4K@60fps x265/H.265 media (generally only 24/30P content of course). I prefer using the `high-quality` profile, unless it happens to come with a major performance hit. HDR appears to be at an experimental state in Gnome, thus won't be enabled in mpv (for now).

**Comment by philipl** [comment]
While 4k24 or 4k30, should work full quality on a 12th gen NUC, no NUC except this year's Lunar Lake, or maybe Meteor Lake, is powerful enough to do 4k60 content with full quality tonemapping. 

So, as long as you're ok with that fact, you should be good to go.
